storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3295,6 +3295,10 @@ static bool rdb_is_ttl_read_filtering_enabled();
 static int rdb_dbug_set_ttl_read_filter_ts();
 #endif
 
+[[nodiscard]] static bool is_autocommit(const THD &thd) {
```

Comment:
could we cache the value for this(to avoid recalculation)? such as add a member variable(such as m_is_autocommit) into Rdb_transaction?

Reply:
We could, but I am not sure it's worth it: the savings would consist of a single AND operation, and maybe of some pointer loads. In all the cases the THD is readily available (no cases of getting it through `current_thd`).